package.json

```diff
@@ -19,7 +19,9 @@
     "stylus": "~0.54.5",
     "uglify-js": "^2.7.5"
   },
-  "devDependencies": {},
+  "devDependencies": {
+    "jsonlint": "^1.6.2"
```

Comment:
> But before it's fixed contributors will see unexpected failures

That's an assumption. These can also happen in micro bumps, when the interface is not changed, but simply linter rules are added in a backwards compatible manner.

When using `^` I am sure you will never look back and check for new major versions. No one does, forcing us to stick with ancient versions. I don't see why anyone would not want the __very__ latest linter ruleset. `>=` does exactly that.